src/app/portlets/dot-edit-page/content/services/dot-edit-content-html/models/dot-contentlets-events.model.ts

```diff
@@ -0,0 +1,28 @@
+import { DotPageContent } from '@portlets/dot-edit-page/shared/models/dot-page-content.model';
+import { DotPageContainer } from '@portlets/dot-edit-page/shared/models/dot-page-container.model';
+
+export interface DotContentletEvent {
+    name: string;
+}
+
```

Comment:
I think maybe we can do by something like:

export interface DotContentletEvent< T > {
    name: string;
    data: T
}

export interface DotContentletEventRelocate extends DotContentletEvent<DotRelocatePayload> {
}

export interface DotContentletEventSave extends DotContentletEvent< DotContentletPayload > {

}